doc/linters.md

```diff
@@ -246,6 +246,28 @@ A regex is also permitted, e.g. to exclude all test namespaces:
 Expected map, found: java.lang.String
 ```
 
+### Bb.edn
+
+*Keyword:* `:bb.edn`
```

Comment:
It was a mistake that I called the :deps.edn linter in this way because officially it's an invalid keyword. 

See:

```
Keywords - Keywords are like symbols, except:

They can and must begin with a colon, e.g. :fred.

They cannot contain '.' in the name part, or name classes.
```

https://clojure.org/reference/reader

This is just a detail, but we'll have to come up with a better name, maybe `:bb-edn`. I'll let you know.

Reply:
OK, more info: https://groups.google.com/g/clojure/c/CCuIp_bZ-ZM/m/9yP41gt7wuoJ
We're good to go with `:bb.edn`.